yaksh/views.py

```diff
@@ -73,13 +74,30 @@ def is_moderator(user):
     if user.groups.filter(name='moderator').exists():
         return True
 
+
 def add_to_group(users):
     """ add users to moderator group """
     group = Group.objects.get(name="moderator")
     for user in users:
         if not is_moderator(user):
             user.groups.add(group)
 
+
+def extract_files(questions_file):
```

Comment:
Is the zip file created in the current directory? If the files are supposed to be discarded at the end then can we create the files in the /tmp/ to avoid permission issue?